<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="drone_3D">
       <attribute name="title">
        <string>3D View</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout" stretch="5,0,0">
          <item>
           <layout class="QVBoxLayout" name="drone_view"/>
          </item>
          <item>
           <widget class="Line" name="line">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QGridLayout" name="gridLayout_2">
            <item row="1" column="0" colspan="3">
             <widget class="QFrame" name="frame">
              <property name="frameShape">
               <enum>QFrame::StyledPanel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Raised</enum>
              </property>
              <layout class="QGridLayout" name="gridLayout_3">
               <item row="2" column="2" colspan="2">
                <widget class="QLabel" name="lblLongValue">
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
               <item row="3" column="2">
                <spacer name="verticalSpacer">
                 <property name="orientation">
                  <enum>Qt::Vertical</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>20</width>
                   <height>40</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item row="1" column="2">
                <widget class="QLabel" name="lblLatValue">
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
               <item row="2" column="0">
                <widget class="QLabel" name="lblLon">
                 <property name="text">
                  <string>y_coord</string>
                 </property>
                </widget>
               </item>
               <item row="4" column="0" colspan="4">
                <widget class="QProgressBar" name="progressBar">
                 <property name="value">
                  <number>24</number>
                 </property>
                </widget>
               </item>
               <item row="0" column="2">
                <spacer name="verticalSpacer_2">
                 <property name="orientation">
                  <enum>Qt::Vertical</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>20</width>
                   <height>40</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item row="1" column="0">
                <widget class="QLabel" name="lblLat">
                 <property name="text">
                  <string>x_coord</string>
                 </property>
                </widget>
               </item>
               <item row="3" column="1">
                <widget class="QLineEdit" name="maxVelText">
                 <property name="placeholderText">
                  <string>1.0</string>
                 </property>
                </widget>
               </item>
               <item row="3" column="0">
                <widget class="QLabel" name="label">
                 <property name="text">
                  <string>Max Vel</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QPushButton" name="btnLaunch">
              <property name="text">
               <string>Launch</string>
              </property>
             </widget>
            </item>
            <item row="4" column="2">
             <widget class="QPushButton" name="btnDown">
              <property name="text">
               <string>DOWN</string>
              </property>
             </widget>
            </item>
            <item row="7" column="1">
             <widget class="QPushButton" name="btnSouth">
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset>
                <normaloff>arrows/down.png</normaloff>arrows/down.png</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>28</width>
                <height>28</height>
               </size>
              </property>
             </widget>
            </item>
            <item row="0" column="0" colspan="3">
             <widget class="QLabel" name="lblFlightMode">
              <property name="text">
               <string> Flight Mode :</string>
              </property>
             </widget>
            </item>
            <item row="4" column="1">
             <widget class="QLabel" name="lblAltValue">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item row="3" column="2">
             <widget class="QPushButton" name="btnUp">
              <property name="text">
               <string>UP</string>
              </property>
             </widget>
            </item>
            <item row="5" column="1">
             <widget class="QPushButton" name="btnNorth">
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset>
                <normaloff>arrows/up.png</normaloff>arrows/up.png</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>28</width>
                <height>28</height>
               </size>
              </property>
             </widget>
            </item>
            <item row="6" column="1">
             <widget class="QPushButton" name="btnRTL">
              <property name="font">
               <font>
                <pointsize>-1</pointsize>
                <weight>75</weight>
                <italic>false</italic>
                <bold>true</bold>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">QPushButton{
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                stop: 0 #D3D3D3, stop: 0.4 #D8D8D8,
                                stop: 0.5 #DDDDDD, stop: 1.0 #E1E1E1);
    border-style: outset;
    border-width: 1px;
    border-radius: 19px;
    border-color: beige;
    font: bold 14px;
    min-width:1em;
    padding: 10px;
}

QPushButton:pressed {
    background-color: rgb(224, 0, 0);
    border-style: inset;
}</string>
              </property>
              <property name="text">
               <string>LAND</string>
              </property>
              <property name="iconSize">
               <size>
                <width>28</width>
                <height>28</height>
               </size>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLabel" name="lblFlightModeValue">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item row="6" column="0">
             <widget class="QPushButton" name="btnWest">
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset>
                <normaloff>arrows/left.png</normaloff>arrows/left.png</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>28</width>
                <height>28</height>
               </size>
              </property>
             </widget>
            </item>
            <item row="6" column="2">
             <widget class="QPushButton" name="btnEast">
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset>
                <normaloff>arrows/right.png</normaloff>arrows/right.png</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>28</width>
                <height>28</height>
               </size>
              </property>
             </widget>
            </item>
            <item row="4" column="0">
             <widget class="QLabel" name="lblAlt">
              <property name="text">
               <string> Altitude    : </string>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QPushButton" name="btnSendTraj">
              <property name="text">
               <string>Send Traj</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="plan_2D">
       <attribute name="title">
        <string>Planning View</string>
       </attribute>
       <widget class="QWidget" name="verticalLayoutWidget">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>781</width>
          <height>501</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="plan_view"/>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
